helm/sealed-secrets/templates/ingress.yaml

```diff
@@ -12,12 +12,16 @@ metadata:
     {{- end }}
   {{- end }}
 spec:
+  {{- if .Values.ingress.tls }}
+  tls:
+  {{ tpl (toYaml .Values.ingress.tls | indent 4) . }}
+  {{- end }}
   {{- if and .Values.ingress.ingressClassName (eq "true" (include "sealed-secrets.supportsIngressClassname" .)) }}
   ingressClassName: {{ .Values.ingress.ingressClassName | quote }}
   {{- end }}
   rules:
-    {{- if .Values.ingress.hostname }}
-    - host: {{ .Values.ingress.hostname }}
+  {{- range .Values.ingress.hosts }}
+    - host: {{ tpl . $ }}
```

Comment:
The `.Values.ingress.hosts` parameter is deprecated, use `.Values.ingress.extraHosts` instead to add additional hostname(s):
- https://github.com/bitnami-labs/sealed-secrets/blob/main/helm/sealed-secrets/values.yaml#L174